proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -49,11 +49,16 @@
         "request.header.NHSD-End-User-Organisation"
     );
 
+    var oneDirectionalSync = context.getVariable(
+        "request.header.NHSD-One-Directional-Sync"
```

Comment:
So the client (NRL) is going to send another header? I thought the custom attribute was going to exist on the app and if a request was made to the app and was allowed that it would be enough?

Just wondering if the logic can stay entirely within the app and if you can access it then it can add a value to the metadata